File: Assignment1/Assignment1/Instructor.cpp
#include "Instructor.h"

// Constructors
Instructor::Instructor(string setFirstName, string setLastName) : User(setFirstName, setLastName) {

}

Instructor::Instructor(string setFirstName, string setLastName, string setWentworthID) : User(setFirstName, setLastName, setWentworthID) {

}

// Methods
void Instructor::searchCourse() {
	cout << "Search Course Method" << endl;
}

void Instructor::showClassList() {
	cout << "Print Class List Method" << endl;
}

void Instructor::showSchedule() {
	cout << "Print Schedule Method" << endl;
}

// Deconstructor
Instructor::~Instructor() {

}

File: Assignment1Java/Instructor.java
package com.Assignment1Java;

public class Instructor extends User {
    // Constructors
    public Instructor(String setFirstName, String setLastName) {
        super(setFirstName, setLastName);
    }

    public Instructor(String setFirstName, String setLastName, String setWentworthID) {
        super(setFirstName, setLastName, setWentworthID);
    }

    // Methods
    public void searchCourse() {
        System.out.println("Instructor Search Course Method");
    }

    public void showClassList() {
        System.out.println("Instructor Print Class List Method");
    }

    public void showSchedule() {
        System.out.println("Instructor Print Schedule Method");
    }
}
